ParraCore/Parra+Analytics.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Parra+Analytics.swift
+//  ParraCore
+//
+//  Created by Mick MacCallum on 12/29/22.
+//  Copyright © 2022 Parra, Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension Parra {
+    /// Logs a new event to the user's current session in Parra Analytics. Events can be used to activate campaigns configured in the Parra dashboard.
```

Comment:
Could we also make a helper here for taking params as Codable so one could make structs for their events just call `logAnalyticsEvent(name, UserCreatedAnalyticsEventBody(user))` or `logAnalyticsEvent(UserCreatedAnalyticsEvent(user))` or even an enum `logAnalyticsEvent(AnalyticsEvent.userCreated(user))` if they wanted to